tests/virt_autotest/login_console.pm

```diff
@@ -26,6 +26,8 @@ sub login_to_console {
     reset_consoles;
     select_console 'sol', await_console => 0;
 
+    my $sut_machine = get_var('SUT_IP', 'nosutip');
+    boot_local_disk_arm_huawei if (check_var('BACKEND', 'ipmi') && check_var('ARCH', 'aarch64') && ($sut_machine =~ /huawei/img));
```

Comment:
Please include `use Utils::Backends 'is_remote_backend';` and replace the check with something simpler, like the following example.

```suggestion
    boot_local_disk_arm_huawei if is_remote_backend && ($sut_machine =~ /huawei_aarch64/);
```

One thing that you have to keep into consideration, is when the SUT, is an actual IP address.... But since this is is a special case for your use case, I have no problem.

Reply:
I will have a look

Reply:
Changes have been made. But I did not change the SUT machine match part. And I think checking on ARCH is necessary.